sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        JobDescription="scikit-learn Inference Basic Recommender Job",
</pre><p>Be consistent with "sklearn" versus "scikit-learn" in descriptions and text. Usually you use "sklearn."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Fixed.